docs/users-guide/13-embedding.md

```diff
@@ -0,0 +1,62 @@
+## Embedding Metabase in other Applications
+
+### Key Concepts
+
+#### Applications
+An important distinction to keep in mind is the difference between Metabase and the embedding application. The charts and dashboards you will be embedding live in the Metabase application, and will be embedded in your (i.e. the embedding application). 
+
+#### Parameters 
+Some dashboards, and questions have the ability to accept parameters. In dashboards, these are synnomous with dashboard filters. For example, if you have a dashboard with a filter on Publisher ID, this can be supplied as a filter when embedding, so that you could insert the dashboard filtered down to a specific Publisher ID.
+
+SQL based questions with template variables can also accept parameters for each variable. So for a query like
+```
+SELECT count(*)
+FROM orders
+WHERE product_id = {{productID}}
+```
+you could specify a specific productID when embedding the question.
+
+#### Signed parameters
+In general, when embedding a chart or dashboard, the server of your embedding application will need to sign a request for that resource. 
+
+If you choose to sign a specific parameter value, that means the user can't modify that, nor is a filter widget displayed for that parameter. For example, if the "Publisher ID" is assigned a value and the request signed, that means the frontend client that renders that dashboard on behalf of a given logged-in user can only see information for that publisher ID. 
```

Comment:
Prefer `front-end` and `back-end` throughout. (You're using both `frontend` and `front end` all willy nilly like. ;)